lms/services/grading_info.py

```diff
@@ -58,8 +58,10 @@ def upsert_from_request(self, request, h_user, lti_user):
         :arg request: A pyramid request
         :arg h_user: The h user this record is associated with
         :type h_user: models.HUser
-        :arg lti_user: The LTI-provided user that this record is associated with
-        :type lti_user: models.LTIUser
+        :arg user_id: The LTI-provided user that this record is associated with
+        :type user_id: str
+        :arg application_instance: The application_instance the user belongs to
+        :type application_instance: application_instance
```

Comment:
```suggestion
        :type application_instance: models.ApplicationInstance
```

I think we're moving away from having type declarations in our docstrings. I don't think we _ever_ really document str types etc.

For the interesting (non basic) types I think these might be nicer as type annotations?

Reply:
I hate typing the docstring type declarations when there already syntax for that in python.  Remove these.

I think something is only "uninteresting" once you know the type, eg `user_id: str` is uninteresting only because you have the annotation there, it could be a integer instead. 

In any case we don't use many type annotations not the PR to discuss this :+1: